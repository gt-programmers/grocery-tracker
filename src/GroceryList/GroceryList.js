import React from 'react';
import './GroceryList.scss';

const GroceryList = () => {
  return (
    <section className="grocery-list">
      <p>GroceryList</p>
    </section>
  )
}

export default GroceryList;