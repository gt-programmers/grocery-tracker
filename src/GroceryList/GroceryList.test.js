import React from 'react';
import GroceryList from './GroceryList';
import { shallow } from 'enzyme';

describe("GroceryList", () => {
  let wrapper = shallow( <GroceryList />)


  it('should match the snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  })
})