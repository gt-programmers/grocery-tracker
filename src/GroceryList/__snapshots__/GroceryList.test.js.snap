// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GroceryList should match the snapshot 1`] = `
<section
  className="grocery-list page"
>
  <p>
    GroceryList
  </p>
</section>
`;
